iXBRLViewerPlugin/viewer/src/js/inspector.js

```diff
@@ -188,11 +190,12 @@ Inspector.prototype.buildHighlightKey = function () {
     var key = this._report.namespaceGroups();
     this._iv.callPluginMethod("extendHighlightKey", key);
 
-    for (var i = 0; i < key.length; i++) {
+    for (const [i, ns] of key.entries()) {
+        const label = this._report.taxonomyNameForURI(this._report.prefixMap()[ns], ns);
```

Comment:
These arguments look reversed?

Reply:
I don't think so.  The first argument is the prefix resolved to a URI.  The second the default, which is the prefix.

I've renamed `ns` to `prefix` to make it clearer what it actually is.

Reply:
Ah, sorry for the misunderstanding, thanks for clarifying.